### PR TITLE
Styles fixes

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
style changes.
A dark mode theme is added.

### CODE REVIEW

File: src/Pages/login/Login.jsx [hunk 1]
```diff
@@ -51,46 +48,47 @@ const Login = () => {
...
               Login
             </button>
-          </div>
+          </Box>
 
-          <div className="form-footer">
+          <Box className="form-footer">
             <p> Don't have an account ?</p>
 
             <Link to="/sign">Create an account</Link>
```
Comment (narharikale) [inline src/Pages/login/Login.jsx:87]: `signin` should be more clear

File: src/features/postSlice.js [hunk 2]
```diff
@@ -94,13 +95,16 @@ export const addToLike = createAsyncThunk("like/add", async (postId) => {
       }
     );
 
+    console.log("ad to like",data);
```
Comment (narharikale) [inline src/features/postSlice.js]: console.log here

File: src/features/postSlice.js [hunk 3]
```diff
@@ -111,9 +115,10 @@ export const removeLike = createAsyncThunk("like/remove", async (postId) => {
       }
     );
 
+    console.log("remove to like",data);
```
Comment (narharikale) [inline src/features/postSlice.js]: here also

**Review by narharikale — APPROVED** [review]
added some suggestions

File: src/Components/Post/Post.css [hunk 4]
```diff
@@ -42,7 +45,7 @@
 
 .icon :hover {
   cursor: pointer;
-  color: var(--primary-color);
+  /* color: var(--primary-color); */
```
Comment (dev-prabhat) [inline src/Components/Post/Post.css]: remove commented code

**Review by dev-prabhat — APPROVED** [review]
Looks good to me 
Just on the mention suggestion

**Pull request merged**